ogpf.f90

```diff
@@ -1733,37 +1733,37 @@ subroutine processcmd(this)
         ! write the plot style for data
         ! this is used only when 3D plots (splot, cplot) is used
         if (allocated(this%txtdatastyle)) then
-            write ( this%file_unit, '(a)' ) 'set style data '//this%txtdatastyle  !set data style
-            write ( this%file_unit, '(a)' )  ! emptyline
+            write ( this%file_unit, '("set style data ", a)' ) this%txtdatastyle
+            write ( this%file_unit, '(a)' )
         end if
 
 
         ! Write options
         if ( this%hasoptions ) then
-            write( unit = this%file_unit, fmt= '(a)') ' '
-            write( unit = this%file_unit, fmt= '(a)') '# options'
-            write( unit = this%file_unit, fmt= '(a)') this%txtoptions
-            write ( this%file_unit, '(a)' )  ! emptyline
+            write ( this%file_unit, '(" ")' )
+            write ( this%file_unit, '("# options")' )
+            write ( this%file_unit, '(a)' ) this%txtoptions
+            write ( this%file_unit, '(a)' )
         end if
 
         ! Check with plot scale: i.e linear, logx, logy, or log xy
-        write( unit = this%file_unit, fmt= '(a)') ' '
-        write( unit = this%file_unit, fmt= '(a)') '# plot scale'
+        write( this%file_unit, '(" ")' )
+        write( this%file_unit, '("# plot scale")' )
         select case (this%plotscale)
             case ('semilogx')
-                write ( this%file_unit, '(a)' ) 'set logscale  x'
+                write ( this%file_unit, '("set logscale  x")' )
             case ('semilogy')
-                write ( this%file_unit, '(a)' ) 'set logscale  y'
+                write ( this%file_unit, '("set logscale  y")' )
             case ('loglog')
-                write ( this%file_unit, '(a)' ) 'set logscale  xy'
+                write ( this%file_unit, '("set logscale  xy")' )
             case default !for no setting
                 !pass
         end select

```

Comment:
Use ' instead " for consistency
Keep inline comments
The reason you deleted fmt=?

Reply:
Same reason as before.